fpga_interchange/examples/chipdb.cmake

```diff
@@ -0,0 +1,134 @@
+function(create_rapidwright_device_db)
+    set(options)
```

Comment:
Please add a documentation block at the top.

Reply:
Done